uint8 mapx
uint8 mapy
---
float32 posx
float32 posy